depthai_sdk/src/depthai_sdk/components/nn_component.py

```diff
@@ -191,33 +227,38 @@ def _parse_model(self, model):
         if model.suffix in ['.blob', '.json']:
             if model.suffix == '.blob':
                 self._blob = dai.OpenVINO.Blob(model.resolve())
-                self._forcedVersion = self._blob.version
+                self._forced_version = self._blob.version
             elif model.suffix == '.json':  # json config file was passed
                 self._parse_config(model)
         else:  # SDK supported model
             models = getSupportedModels(printModels=False)
-            if str(model) not in models:
+            zoo_models = blobconverter.zoo_list()
+
+            if str(model) in models:
+                model = models[str(model)] / 'config.json'
+                self._parse_config(model)
+            elif str(model) in zoo_models:
```

Comment:
The problem with going with blobconverter is that we don't know anything about the model; label map, decoding logic, metadata (eg yolo), etc. so we have to make sure to throw useful error if these aren't provided to SDK via some other api.

Reply:
Added warning